plugins/utils/mini.nix

```diff
@@ -39,12 +36,12 @@ in
     };
   };
 
-  config = mkIf cfg.enable {
-    extraPlugins = [ cfg.package ];
-
-    extraConfigLua = concatLines (
-      mapAttrsToList (
-        name: config: "require('mini.${name}').setup(${helpers.toLuaObject config})"
+  # NOTE: We handle each module explicitly
+  callSetup = false;
+  extraConfig = cfg: {
```

Comment:
I think we also need `hasSettings = false`

Reply:
I would prefer to keep it mkNeovimPlugin strictly for accuracy and reporting with `list-plugins`, even if we dont use all the inherited functionality.